pynxtools/eln_mapper/README.md

```diff
@@ -1,10 +1,11 @@
 # ELN generator
-This is a helper tool for generating eln
-- The simple eln generator that can be used in a console or jupyter-notebook
-- Scheme based eln generator that can be used in NOMAD and the eln can be used as a custom scheme in NOMAD.
+This is a helper tool for generating ELN files that can be used to add metadata to the dataconverter routine.
+Two types of ELN are supported (by passing the flag `eln-type`):
+- **eln**: The simple ELN generator that can be used in a console or jupyter-notebook.
+- **scheme_eln**: Scheme based ELN generator that can be used in NOMAD and the ELN can be used as a custom scheme in NOMAD.
 
 ```
-$ eln_generator --options <value>
+$ generate_eln --options <value>
```

Comment:
Can we change this to the --help command? It will then show the output below shown in the README.
```suggestion
$ generate_eln --help
```

Reply:
Done and updated the help in click.